pages/linux/tomb.md

```diff
@@ -0,0 +1,26 @@
+# tomb
+
+> Manage the creation and access of encrypted storage files.
```

Comment:
We should mention that `tomb` creates an encrypted "folder" to store files. From your description, it might appear that `tomb` creates encrypted files too.